documentation-website/language/it.yml

```diff
@@ -43,7 +43,7 @@ socials:
     title: "idrinth/api-bench su Medium"
     alt: "Medium"
   youtube:
-    title: "idrinth-api-bench su YouTube"
+    title: "idrinth su YouTube"
```

Comment:
the youtube name is already the one on the left, please revert the change

Reply:
Reverted